src/v2/Apps/Artist/Routes/Consign/Components/ArtistConsignHeader/ArtistConsignHeaderImages.tsx

```diff
@@ -83,7 +86,16 @@ const LeftImagePhoto: React.FC<ImageProps> = ({ image }) => {
 }
 
 const RightImagePhoto: React.FC<ImageProps> = ({ image }) => {
-  return <ResponsiveImage src={image.resized.url} />
+  return (
+    <div
+      style={{
+        width: "100%",
+        paddingBottom: "100%",
+        backgroundImage: `url(${image.resized.url})`,
+        backgroundSize: "contain",
+      }}
+    />
+  )
```

Comment:
@dzucconi - this would really benefit from using `ResponsiveBox` or at least creating a component that wraps this

Reply:
I mean sure; we shouldn't do this _at all_. I'm simply replacing it with what was there previously. This is just exactly what `ResponsiveImage` was doing, not what it should be.